paulonia/processes/kanban.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Kanban
+pagination:
+    next:
+        link: ../vacation
+        name: Vacaciones
+    previous:
+        link: ../
+        name: Introducción
+---
+
+Es el principal marco de trabajo de desarrollo ágil que utilizamos en Paulonia. Antes usabamos Scrum, pero con el pasar del tiempo nos dimos cuenta de que no encajaba con nuestro ritmo de trabajo, sin embargo, se han sacado ideas de Scrum para integrarlas a nuestro nuevo marco de trabajo junto con Kanban.
+
+# Tablero y Tareas
+
+El tablero es Kanban y la prioridad de las tareas tiene mucho peso aquí. La razón por la que usamos Kanban en vez de Scrum es que nuestros clientes y trabajos son volátiles provocando que entren tareas en medio del sprint o que las prioridades de las tareas cambien constantemente, lo que rompía toda la armonía que Scrum dicta.
+
+Kanban nos permite crear tareas para que puedan ser realizadas en cualquier momento cambiando las prioridades de las tareas existentes.
+
+## Estructura del tablero
+
+- **Backlog:** Una tarea recién creada va a esta columna. Es un listado de tareas creadas que no tienen la prioridad o aún no han sido seleccionadas para ser realizadas.
+- **Selected:** Aquí van las tareas que han sido seleccionadas para ser realizadas.
+- **Recurring:** Son tareas que se hacen en diferentes periodos y no tienen un tiempo de fin definido. Ej. Realizar las boletas de pago mensuales.
+- **Task for the day:** Son tareas que han sido seleccionadas por el [Assignee](#assignee) para ser las que va a realizar a continuación on en el día.
+- **In progress:** Son las tareas que se están realizando actualmente.
+- **Waiting for:** Son las tareas bloqueadas. Una tarea puede estar bloqueada, por ejemplo, porque es necesaria la intervención de otra persona, es necesaria que se termine otra tarea, ha entrado otra tarea mucho más prioritaria, etc.
```

Comment:
Especificar que cada persona no pede tener mas de 3 tareas en esta columna.

Reply:
Dale :+1:

Reply:
Lo voy a colocar en la columna In progres y en la columna de Waiting for